runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsMinimapOverlay.java

```diff
@@ -61,15 +73,60 @@ public Dimension render(Graphics2D graphics)
 
 	private void renderPlayerOverlay(Graphics2D graphics, Player actor, Color color)
 	{
-		final String name = actor.getName().replace('\u00A0', ' ');
-
 		if (config.drawMinimapNames())
 		{
-			final net.runelite.api.Point minimapLocation = actor.getMinimapLocation();
+			String name = actor.getName().replace('\u00A0', ' ');
+			String tag = "";
+			String prefix = "tag_";
+			if (FriendTaggingPlugin.taggedFriends.containsKey(prefix + name.trim().toLowerCase()))
+			{
+				tag = " [" + FriendTaggingPlugin.taggedFriends.get(prefix + name.trim().toLowerCase()) + "] ";
+			}
+
+			name += tag;
+
+			net.runelite.api.Point minimapLocation = actor.getMinimapLocation();
 
 			if (minimapLocation != null)
 			{
-				OverlayUtil.renderTextLocation(graphics, minimapLocation, name, color);
+				if (config.showCombatLevel())
+				{
+					if (config.showCombatLevel())
+					{
+						name += "-(" + actor.getCombatLevel() + ")";
+					}
+				}
+				if (config.drawMinimapNames())
+				{
+					if (actor.getSkullIcon() != null && config.playerSkull())
+					{
+						switch (actor.getSkullIcon())
+						{
+							case SKULL:
```

Comment:
This should probably be an if statement, unless you feel like adding dmm skulls as well